google-assistant-webserver/README.md

```diff
@@ -0,0 +1,39 @@
+Google Assistant Webserver
+--------------------------
+About
+--------------------------
+A virtual personal assistant developed by Google (text input via webserver). Once running and authenticated, you can send a GET request to http://hassio.local:5000/broadcast_message?message=[MESSAGE] to broadcast a text message to all your Google Assistants.
+
```

Comment:
I suggest also adding:
You can send a GET request to http://[your.hassio.IP]:5000/command?message=[google_command] to send a command to the Google Assistant.